.github/CODEOWNERS

```diff
@@ -0,0 +1,31 @@
+# Package code owners
+
+# The listed owners will be automatically added as reviewers for PRs,
+# to ensure code quality and consistency of the package, and identify
+# possible side effects.
+# PRs should still be peer-reviewed by the team opening the PR
+
+# See https://help.github.com/articles/about-codeowners/ for syntax
+
+/pkg/metadata/kubernetes/               @DataDog/burrito
+/pkg/metadata/ecs/                      @DataDog/burrito
+
+/pkg/metrics/percentile/                @DataDog/sandwich
+
+/Dockerfiles/                           @DataDog/kenafeh
+/pkg/tagger/                            @DataDog/kenafeh
+/pkg/util/docker/                       @DataDog/kenafeh @DataDog/burrito
+/pkg/util/ecs/                          @DataDog/kenafeh @DataDog/burrito
+/pkg/util/kubernetes/                   @DataDog/kenafeh @DataDog/burrito
+/pkg/collector/listeners/docker*        @DataDog/kenafeh
+/pkg/collector/corechecks/containers/   @DataDog/kenafeh
+
+/cmd/                                   @DataDog/croissant
+/pkg/aggregator/                        @DataDog/croissant
+/pkg/collector/                         @DataDog/croissant
+/pkg/forwarder/                         @DataDog/croissant
+/pkg/metadata/                          @DataDog/croissant
```

Comment:
Same here we should own ecs and kubernetes.

Reply:
/pkg/metadata/ecs | kubernetes are owned by burito at the top of the files. These just serialize the information to a burito-specific playload. The connection/parsing logic is in /pkg/util/ecs | kubernetes which both teams own